### PR TITLE
Push the images to test namespace from dev after e2e test passes

### PR DESCRIPTION
**Pull request by Kinshuk1993**
Push the images to test namespace from dev after e2e test passes

### CODE REVIEW

File: products/bash/deploy-og-sub.sh [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+#   Overriding the namespace
+#     ./deploy-og-sub -n cp4i-prod
+#
+# function usage {
+#     echo "Usage: $0 -n <namespace>"
+# }
+#
+namespace="cp4i"
+SCRIPT_DIR="$(dirname $0)"
+echo "Current Dir: $SCRIPT_DIR"
```
Comment (DanRoseus) [inline products/bash/deploy-og-sub.sh:31]: These are not used, remove?

File: DrivewayDentDeletion/Operators/prereqs.sh [hunk 2]
```diff
@@ -105,37 +135,57 @@ else
...
+
+for image_project in "${image_projects[@]}"
+do
+  echo "INFO: Creating secret to pull images from the ER"
+
+  oc create -n ${image_project} secret docker-registry ibm-entitlement-key --docker-server=${ER_REGISTRY} \
+    --docker-username=${ER_USERNAME} --docker-password=${ER_PASSWORD} -o yaml | oc apply -f -
+
+  echo "INFO: Creating operator group and subscription in ${image_project}"
+  ${PWD}/../../products/bash/deploy-og-sub.sh -n ${image_project}
```
Comment (DanRoseus) [inline DrivewayDentDeletion/Operators/prereqs.sh:184]: It looks like these won't exit if the child script fails?

File: DrivewayDentDeletion/Operators/cicd-webhook-triggers.yaml [hunk 3]
```diff
@@ -85,27 +137,36 @@ spec:
...
-      script: "/workspace/git-source/DrivewayDentDeletion/Operators/test-api-e2e.sh -n {{NAMESPACE}} -t $(params.imageTag)"
+    - name: pull-retag-push-to-test
+      image: quay.io/buildah/stable:latest
+      securityContext:
+        runAsUser: 0
+        privileged: true
+      command:
+        - sh
+        - -c
+        - "buildah pull --tls-verify=false image-registry.openshift-image-registry.svc:5000/{{NAMESPACE}}-ddd-dev/$(params.imageName):$(params.imageTag) && \
```
[the remark this answers is not in the document]
Comment (Kinshuk1993) [reply DrivewayDentDeletion/Operators/cicd-webhook-triggers.yaml:166]: As discussed, we would use the PVC that is used with the build task and code change will be a part of another PR

File: DrivewayDentDeletion/Operators/prereqs.sh (hunk 2 above)
Comment (Kinshuk1993) [reply DrivewayDentDeletion/Operators/prereqs.sh:184]: There is an exit put in place if the child scripts fail

File: products/bash/configure-postgres.sh [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  echo "INFO: Table already exists, skipping this step"
+fi
+
+if ! oc exec -n postgres -it $(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}') \
+-- psql -U ${namespace} -d db_${namespace} -c '\dt'  | grep -i quotes ; then
+
+echo "INFO: Creating tables in the database db_${namespace}"
+  oc exec -n postgres -it $(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}') \
+    -- psql -U ${namespace} -d db_${namespace} -c \
+  'CREATE TABLE QUOTES (
```
[the remark this answers is not in the document]
Comment (Kinshuk1993) [reply products/bash/configure-postgres.sh:47]: Done

File: products/bash/deploy-og-sub.sh (hunk 1 above)
Comment (Kinshuk1993) [reply products/bash/deploy-og-sub.sh:31]: Done

File: DrivewayDentDeletion/Operators/prereqs.sh [hunk 5]
```diff
@@ -105,37 +135,57 @@ else
...
+  sleep 10
+  echo "INFO: Releasing Navigator in ${image_project}"
+  ${PWD}/../../products/bash/release-navigator.sh -n ${image_project} -r ${nav_replicas}
+  echo "INFO: Releasing ACE dashboard in ${image_project}"
+  ${PWD}/../../products/bash/release-ace.sh -n ${image_project}
```
[the remark this answers is not in the document]
Comment (Kinshuk1993) [reply DrivewayDentDeletion/Operators/prereqs.sh:189]: Done

File: products/bash/configure-postgres.sh [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+EOF
+else
+  echo "INFO: Table already exists, skipping this step"
+fi
+
+if ! oc exec -n postgres -it $(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}') \
+-- psql -U ${namespace} -d db_${namespace} -c '\dt'  | grep -i quotes ; then
+
+echo "INFO: Creating tables in the database db_${namespace}"
+  oc exec -n postgres -it $(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}') \
```
Comment (DanRoseus) [inline products/bash/configure-postgres.sh]: I think only one oc exec should be used, the 2nd one, with the if checking the result of the 2nd one

File: products/bash/release-ace.sh [hunk 7]
```diff
@@ -16,6 +16,7 @@
 #   -n : <namespace> (string), Defaults to "cp4i"
 #   -r : <dashboard-release-name> (string), Defaults to "ace-dashboard-demo"
 #   -e : <designer-release-name> (string), Defaults to "ace-designer-demo"
+#   -c : <releaseBoth> (string), Default to releasing both ace dashboard and designer
```
Comment (DanRoseus) [inline products/bash/release-ace.sh:19]: This options sounds very confusing. So I think if not specified it releases both dashboard and designer, and if specified it releases only dashboard? That sounds very confusing. We don't want to add confusing options and then have to support them from now on

File: products/bash/configure-postgres.sh (hunk 6 above)
Comment (Kinshuk1993) [reply products/bash/configure-postgres.sh]: Removed the second if check and added IF NOT EXISTS

File: products/bash/release-ace.sh (hunk 7 above)
Comment (Kinshuk1993) [reply products/bash/release-ace.sh]: Split the release ace into two scripts and called both in release ace and release dashboard from the prereqs for the tekton pipelines.

**Pull request merged**